### PR TITLE
more navigation links, additional info and small corrections

### PR DESCRIPTION
**Pull request by damianloew**
JIRA: PD-205

### CODE REVIEW

File: ports/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Ports
+
+Open source tools adapted to Phoenix-RTOS are called ports. A source code of each port is downloaded from its official website as an archive file.
```
Comment (niewim19) [inline ports/README.md:3]:
```suggestion
Open-source tools adapted to Phoenix-RTOS are called ports. A source code of each port is downloaded from its official website as an archive file.
```

File: tests/README.md [hunk 2]
```diff
@@ -1 +1,57 @@
 # Testing process
+
+The testing process uses a Phoenix-RTOS testing framework implemented in Python. The framework provides an environment for running both unit and functional tests.
```
Comment (niewim19) [inline tests/README.md:3]:
```suggestion
The testing process uses a Phoenix-RTOS testing framework written in Python. The framework provides an environment for running both unit and functional tests.
```

File: tests/README.md [hunk 3]
```diff
@@ -1 +1,57 @@
...
+```bash
+python3 runner.py --help
+```
+
+<img src="_images/runner-help.png" width="600px">
+
+## Using test runner via Github Actions
+
+Creating pull requests and pushing to master or feature/* branches of `phoenix-rtos` Github repositories triggers Github Actions in order to run tests.
+Actions in a pull request created by some Github user has to be approved for the first time.
```
Comment (niewim19) [inline tests/README.md:47]:
```suggestion
When a new contributor creates first pull request it has to be approved before actions run.
```

**Review by niewim19 — COMMENTED** [review]
nit picks

File: tests/README.md (hunk 3 above)
Comment (damianloew) [reply tests/README.md]: Thanks, changes applied

**Review by niewim19 — APPROVED** [review]
lgtm

**Pull request merged**